Document tokenized From addresses for Emails (#727)
mautic/user-documentation: docs/configuration/settings.rst

  :width: 600
  :alt: Screenshot showing Mail Send Settings Configuration in Mautic

> * **Name to send mail as** - The default name Emails come from. This is typically something like ``{YourCompany Marketing Team}`` or ``{YourCompany}``. You can also use Contact field tokens like ``{contactfield=companyname|Default Name}`` to personalize the sender name per Contact.

Reviewer: Avoid duplication.

```suggestion
* **Name to send mail as** - The default name Emails come from. This is typically something like ``{YourCompany Marketing Team}`` or ``{YourCompany}``. You can also use Contact field tokens such as ``{contactfield=companyname|Default Name}`` to personalize the sender name per Contact.
```

* **Mailer is owner** - If Contacts in Mautic have owners, select Yes to use the Contact owner as the sender of Emails to any Contacts they're listed as the owner for.

   .. note::

>       When enabled, **Mailer is owner** sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's **Advanced** tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :ref:`mailer as owner` and :ref:`Sender resolution hierarchy<sender resolution hierarchy>`.

Reviewer: Because this is an unordered list, indentation should be 2 spaces instead of 3. Also, we can simply link to the custom target and no need to have custom text.
 
```suggestion
  .. note::

     When enabled, **Mailer is owner** sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's **Advanced** tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :ref:`mailer as owner` and :ref:`sender resolution hierarchy`.
```

@promptless-for-oss please address this suggestion

>       When enabled, **Mailer is owner** sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's **Advanced** tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :ref:`mailer as owner`. For the complete sender resolution hierarchy, see :ref:`Sender resolution hierarchy<sender resolution hierarchy>`.

Reviewer: @promptless-for-oss I think we can just combine the links in one sentence.

```suggestion
      When enabled, **Mailer is owner** sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's **Advanced** tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :ref:`mailer as owner` and :ref:`Sender resolution hierarchy<sender resolution hierarchy>`.
```

>     When enabled, Mailer is owner sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's Advanced tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information see :doc:`Mailer is owner</channels/emails>`. For the complete sender resolution hierarchy, see :ref:`Sender resolution hierarchy<sender-resolution-hierarchy>`.

Reviewer: ```suggestion
    When enabled, Mailer is owner sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's Advanced tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :doc:`Mailer is owner</channels/emails>`. For the complete sender resolution hierarchy, see :ref:`Sender resolution hierarchy<sender-resolution-hierarchy>`.
```

>     When enabled, Mailer is owner sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's Advanced tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :doc:`Mailer is owner</channels/emails>`. For the complete sender resolution hierarchy, see :ref:`Sender resolution hierarchy<sender-resolution-hierarchy>`.

Reviewer: There are a couple of things:

- The note admonition is part of the bullet point. Adjust the indentations.
- Link directly to Mailer as Owner

```suggestion
   .. note::

      When enabled, **Mailer is owner** sends Emails from the Contact's owner, overriding system defaults and plain Email From addresses. However, tokenized From addresses on an Email's **Advanced** tab take precedence over the owner sender when the token resolves to a valid value. Every Contact owner's domain must have ``SPF`` and ``DKIM`` records. You can configure this setting for individual Emails, rather than globally. For more information, see :ref:`mailer as owner`. For the complete sender resolution hierarchy, see :ref:`Sender resolution hierarchy<sender resolution hierarchy>`.
```

Reviewer: @promptless-for-oss please address this suggestion
